docs/content/theme/styles/hedgedoc-custom.css

```diff
@@ -12,3 +12,6 @@
     --md-accent-bg-color: hsla(0, 0%, 100%, 1);
     --md-accent-bg-color--light: hsla(0, 0%, 100%, 0.7);
 }
+.md-grid {
+  max-width: 1440px;
```

Comment:
The theme default is `61rem`, maybe `53rem` work fine?

Reply:
This is intentional. I'm more of a fan of fixed widths instead of "about x characters".

Reply:
Especially because we have a non-mono font